main.py

```diff
@@ -0,0 +1,52 @@
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import col, split, row_number, explode
+from pyspark.sql import Window
+
+if __name__ == '__main__':
+    spark = SparkSession \
+        .builder \
+        .master("local[12]") \
+        .appName("main") \
+        .getOrCreate()
+
+    """
+    The First Task'a
+    """
```

Comment:
doc-string should be after func definition, if you want to put comments inside function use `#`